### DISCUSSION TITLE
Access for Educators - Also seeing issues with proof of status

### DISCUSSION THREAD

**Discussion by rpigu-i** [community/community#55050] [GitHub Education]
### Hi everyone,

Hi GitHub,

Is it possible you could add a more robust mechanism to prove academic/teacher status? Currently when applying you are asked to take a photo as a for of proof of teacher/tutor status. Many of us who tutor remotely for universities don't have ID cards or faculty email. What we do have are contracts showing our employment terms with the higher education institution. 

If you could include a mechanism to upload a copy of the contract, this would be easier than trying to screen shot. I've tried several times and my application has been rejected. Note, I've been working with this institution for the past 3 years and was also a student in the past and not had issues previously.

I see some students are having a similar issue when applying for student benefits: https://github.com/orgs/community/discussions/53695 

Thanks!

**Comment by samus-aran** [comment]
Hi @rpigu-i ! 👋 

First off I want to thank you for sharing your experiences and that we absolutely understand your frustration. We are trying our best to continually improve our system to be as easy as possible for legitimate students everywhere to verify their academic status, while also ensuring that bad actors impersonating students don't jeopardise our ability to provide academic benefits of value.

As you can imagine there are many types of academic verification and our system is constantly evolving to accommodate this without setting the bar too high. However sometimes we may require a little more evidence from the user applying and we thank you for being patient.

We do except other forms of evidence for the faculty verification but due to the high risk of image manipulation we request both students and faculty to submit a live photo of their evidence rather than an upload. We understand that users may not have access to high-quality cameras (laptop or phones) but as long as the text is legible the system should be able to process it. Here are a few tips on using a photo to capture.

1. Ensure your image (photo or upload) is legible. If you are unable to read its text on the image itself then the verification process may not be able to either. If you are taking a photo of a screen try to have it as large a resolution as possible - take a photo of the evidence on a monitor (a laptop perhaps - or a TV) rather than a phone.
2. If you can, print off your evidence and take a photo of this copy.
3. Keep the text legible. It's preferred to have the full shot of the evidence but try and capture the important sections of it to confirm your student status. If the photo is too large and makes the text tiny then zoom in appropriately.

I would also recommend that you ensure your evidence correspond to details in your GitHub profile. The verification process looks at your GitHub username, email and name ( as well as other points) to cross reference the evidence submitted. Do you use a variation of your name or a nickname?  It’s these small but important points to consider when applying that can make a much smoother process.

If you need help with applying please check out our [Apply to GitHub Global Campus for Students](https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/github-global-campus-for-students/apply-to-github-global-campus-as-a-student) as well as our [Why wasn't my application to Global Campus for students approved?](https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/github-global-campus-for-students/why-wasnt-my-application-to-global-campus-for-students-approved). We aim to assist our students by providing rejection reasons throughout the application process so that you can take steps to rectify and re-apply. But we also understand that there are times where you need a little more guidance and we're here to nudge you in the right direction. Best of luck!
